tests/savepoint/translate/translate_d_sw.py

```diff
@@ -66,7 +66,9 @@ def ubke(
     dt5: float,
 ):
     with computation(PARALLEL), interval(...):
-        ub = d_sw.ubke(uc, vc, cosa, rsina, ut, ub, dt4, dt5)
+        dt = 2.0 * dt5
+        ub, _ = d_sw.interpolate_uc_vc_to_cell_corners(uc, vc, cosa, rsina, ut, ut)
+        ub = ub * dt
```

Comment:
Yes, we want to make sure the tests are testing the stencil code, and not the test code. In this case the math isn't duplicated, it's only done here. This needed to be updated when I refactored ubke to take in only one timestep, and to return wind without the temperature folded in. The line before and after here are wrappers which restore the old behavior (since that's what the Fortran code provides data for).

Reply:
ok true, it's not technically duplicated. I usually try to keep compute code out of the test area to make investigating errors easier, but with the changes made, we probably need to do this. so leave as-is to preserve the test functionality.